### ISSUE TITLE
error when adding some workflows

### ISSUE THREAD

**Issue by MingChen0919** [tripal/tripal_galaxy#36]
when adding these two workflows:
```
https://raw.githubusercontent.com/statonlab/dibbs/master/workflow-updates/RNASeq-counts_(cuffquant%2C_paired_end).ga
https://raw.githubusercontent.com/statonlab/dibbs/master/workflow-updates/RNASeq-counts_(cuffquant%2C_single_end).ga
```

I got this error:

```
Notice: Undefined index: name in tripal_galaxy_build_webform() (line 97 of /data/www/html/hardwoods_dev/drupal-7.52/sites/all/modules/custom/tripal_galaxy/includes/tripal_galaxy.webform.inc).
Notice: Undefined index: name in tripal_galaxy_build_webform() (line 99 of /data/www/html/hardwoods_dev/drupal-7.52/sites/all/modules/custom/tripal_galaxy/includes/tripal_galaxy.webform.inc).
```

**Comment by cwytko** [comment]
May you try using these workflows again, I think this has been patched.
I tested both on my instance (Galaxy 16.04) and both workflows are being accepted and generated properly

**Comment by MingChen0919** [comment]
Unfortunately, I still got the same error, I imported the workflow from the WSU galaxy. Do you have access to the WSU galaxy instance?
https://galaxy.bioinfo.wsu.edu/u/mingchen0919/w/rnaseq-normalized-counts-cuffnorm
https://galaxy.bioinfo.wsu.edu/u/mingchen0919/w/rnaseq-normalized-counts-cuffnorm-single-end

**Comment by cwytko** [comment]
@spficklin Should we check with the Main lab to see if our most recent code is pulled and working on the WSU server?

**Comment by spficklin** [comment]
Issues #35 and #36 seem to be duplicates.  I'm marking this one closed as I posted the fix on issue #35
